Content.Server/Silicons/Borgs/BorgSystem.cs

```diff
@@ -344,6 +371,23 @@ private void UpdateBatteryAlert(Entity<BorgChassisComponent> ent, PowerCellSlotC
     /// </summary>
     public void BorgActivate(EntityUid uid, BorgChassisComponent component)
     {
+        // Goobstation set pOrg name if a pAI wakes up inside it
+        string? pAIName = null;
+        if (!_container.TryGetContainer(uid, component.BrainContainerId, out var brainContainer))
+            return;
+        foreach (var containedEntity in brainContainer.ContainedEntities)
+        {
+            if (!TryComp<PAIComponent>(containedEntity, out var paiComponent))
+                continue;
+            if (paiComponent.LastUser != null)
+            {
+                var userName = Name(paiComponent.LastUser.Value);
+                if (!string.IsNullOrWhiteSpace(userName))
+                    pAIName = $" ({userName}'s pAI)";
```

Comment:
Same here
```suggestion
                    pAIName = $"({userName}'s pAI)";
```
TBH I am looking at it rn, and use 
`Loc.GetString("pai-system-pai-name", ("owner", component.LastUser));` instead? No need to reinvent the wheel

Reply:
I'm not sure exactly how that works but that might actually fix another semi-known bug I've been struggling with if i figure out how that works

Reply:
done